### DISCUSSION TITLE
UEFI capsule updates not available or enabled in firmware setup

### DISCUSSION THREAD

**Discussion by bert269** [fwupd/fwupd#5999] [General]
Trying to do the latest ubuntu (22.04) updates to Orangepi 5 and received this error:

![image](https://github.com/fwupd/fwupd/assets/18601216/a796c004-8a9b-41ea-bb0f-05682c7ffcae)

**Comment by superm1** [comment]
So why do you think it's a bug?

**Comment by bert269** [comment]
because it was working fine until the latest software updates and did not receive this message.
